octopusdeploy/resource_project_scheduled_trigger.go

```diff
@@ -0,0 +1,99 @@
+package octopusdeploy
+
+import (
+	"context"
+	"github.com/OctopusDeploy/go-octopusdeploy/v2/pkg/client"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"log"
+)
+
+func resourceProjectScheduledTrigger() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceProjectScheduledTriggerCreate,
+		DeleteContext: resourceProjectScheduledTriggerDelete,
+		Importer:      getImporter(),
+		ReadContext:   resourceProjectScheduledTriggerRead,
+		Schema:        getProjectScheduledTriggerSchema(),
+		UpdateContext: resourceProjectScheduledTriggerUpdate,
+	}
+}
+
+func resourceProjectScheduledTriggerRead(ctx context.Context, d *schema.ResourceData, m interface{}) diag.Diagnostics {
+	client := m.(*client.Client)
+
+	scheduledTrigger, _ := client.ProjectTriggers.GetByID(d.Id())
```

Comment:
This can also return an error. If err is not nil then it's probably best to `return diag.FromErr(err)`